templates/plugin/python/module.conf.erb

```diff
@@ -1,7 +1,11 @@
   Import "<%= @module %>"
 
   <Module "<%= @module %>">
-  <% @config.sort.each do |key,value| -%>
-    <%= key -%> <%= value -%>
-  <% end %>
+  <%- @config.sort.each do |key,value| -%>
+  <%- if value.is_a?(String) -%>
+    <%= key %> "<%= value %>"
```

Comment:
Be careful, this doesn't check if string is already quoted. Have a look at the [specs](https://github.com/pdxcat/puppet-module-collectd/blob/master/spec/classes/collectd_plugin_python_spec.rb#L90).